.github/scripts/main/main.sh

```diff
@@ -172,7 +172,7 @@ test_project () {
   project=$2
```

Comment:
url should be removed from the parameters since it's not used anymore
```suggestion
  project=$1
```